src/types.mligo

```diff
@@ -94,10 +102,11 @@ type proposal =
   // ^ address of the proposer
   ; proposer_frozen_token : nat
   // ^ amount of frozen tokens used by the proposer, exluding the fixed fee
-  ; voters : voter_map
-  // ^ voter data
   ; quorum_threshold: quorum_threshold
   // ^ quorum threshold at the cycle in which proposal was raised
+  ; is_proposer_unfreezed: bool
```

Comment:
If this indicate that the proposal is either flushed or dropped, then I think it should be named accordingly.